plugins/jvm-abi-gen/src/org/jetbrains/kotlin/jvm/abi/JvmAbiOutputExtension.kt

```diff
@@ -53,6 +53,11 @@ class JvmAbiOutputExtension(
         val outputFiles: OutputFileCollection,
         val removeDebugInfo: Boolean,
     ) : OutputFileCollection {
+
+        private val classesToBeDeleted = abiClassInfos.mapNotNullTo(mutableSetOf()) { (className, action) ->
```

Comment:
Please inline this variable

Reply:
I assume that you ask for that because of better readability and cognitive scope management. 

I can do it, but originally I put that property that high because I wanted to compute a set of removed classes only once per `jvm-abi-gen` invocation If I inline the property it will be computed for every class in the resulting jar which could be possibly a lot.

As a compromise I made it immutable so nobody think that it is being changed elsewhere. If I am wrong and there is another reason for inlining - please let me know!

Reply:
You're right, thanks for the explanation!